src/VPolytope.jl

```diff
@@ -101,6 +102,77 @@ function σ(d::AbstractVector{N},
     end
 end
 
+"""
+    ∈(x::AbstractVector{N}, P::VPolytope{N};
+      solver=GLPKSolverLP(method=:Simplex))::Bool where {N<:Real}
+
+Check whether a given point is contained in a polytope in vertex representation.
+
+### Input
+
+- `x` -- point/vector
+- `P` -- polytope in vertex representation
+
+### Output
+
+`true` iff ``x ∈ P``.
+
+### Algorithm
+
+We check, using linear programming, the definition of a convex polytope that a
+point is in the set if and only if it is a convex combination of the vertices.
+
+Let ``v_j`` be the ``m`` vertices of `P`.
+Then we solve the following ``m``-dimensional linear program.
+
+```math
+\\max 0 \\text{s.t.}
+\bigwedge_{i=1}^n \\sum_{j=1}^m λ_j v_j[i] = x[i]
+\\sum_{j=1}^m λ_j = 1
+\bigwedge_{j=1}^m λ_j ≥ 0
+```
+"""
+function ∈(x::AbstractVector{N}, P::VPolytope{N};
+           solver=GLPKSolverLP(method=:Simplex))::Bool where {N<:Real}
+    vertices = P.vertices
+
+    # special cases: 0 or 1 vertex
+    if length(vertices) == 0
+        return false
+    elseif length(vertices) == 1
+        return x == vertices[1]
+    end
+
+    @assert length(x) == dim(P)
+
+    m, n = length(vertices), length(x)
```

Comment:
simplification: `length(x)` is used a couple of times above, you could move it on top 

```julia
isempty(P) && return false
vertices = P.vertices
m, n = length(vertices), length(first(vertices))
...
```